Lib/fontTools/varLib/mutator.py

```diff
@@ -154,6 +154,14 @@ def instantiateVariableFont(varfont, location, inplace=False):
 
 	By default, a new TTFont object is returned. If ``inplace`` is True, the
 	input varfont is modified and reduced to a static font.
+
+	When the set_macos_overlap_rendering_bit parameter is defined as True, the bit 6
```

Comment:
I would make this description much shorter and simply link to the OT spec for OVERLAP_SIMPLE flag

Reply:
Ok, no problem. I will make this change

Reply:
https://github.com/fonttools/fonttools/pull/1518/commits/fee4848d4ae1ee0d69d368cc4d731423172111b3